src/Core/Vault/Services/Implementations/CipherService.cs

```diff
@@ -678,15 +678,14 @@ await _collectionCipherRepository.UpdateCollectionsForAdminAsync(cipher.Id,
         await _pushService.PushSyncCipherUpdateAsync(cipher, collectionIds);
     }
 
-    public async Task ImportCiphersAsync(
+    public async Task ImportFolderCiphersAsync(
```

Comment:
Note to self: These should likely be moved into a tools-owned import service leveraging methods from the CipherService/Repository